apps/collabora.sh

```diff
@@ -5,11 +5,7 @@
 # shellcheck disable=2034,2059
 true
```

Comment:
no need to change collabora, or onlyoffice.

Reply:
what shall I do here? change this to `false`?

Reply:
Or did you meant changing those scripts at all?

Reply:
> Or did you meant changing those scripts at all?

Yes, I think the function already is good as is.